internal/internal_task_handlers_test.go

```diff
@@ -1255,10 +1255,12 @@ func (t *TaskHandlersTestSuite) TestLocalActivityRetry_WorkflowTaskHeartbeatFail
 	workflowTaskStartedEvent := createTestEventWorkflowTaskStarted(3)
 	now := time.Now()
 	workflowTaskStartedEvent.EventTime = &now
+	// Task timeout must be larger than local activity retry backoff
+	wftTimeout := 2 * time.Second
```

Comment:
I'm not sure I understand the change. It should work with the task timeout being shorter due to task heartbeat.